StreamingKit/StreamingKit/STKQueueEntry.m

```diff
@@ -45,7 +45,7 @@ -(double) calculatedBitRate
 	{
 		if (processedPacketsCount > STK_BIT_RATE_ESTIMATION_MIN_PACKETS_PREFERRED || (audioStreamBasicDescription.mBytesPerFrame == 0 && processedPacketsCount > STK_BIT_RATE_ESTIMATION_MIN_PACKETS_MIN))
 		{
-			double averagePacketByteSize = processedPacketsSizeTotal / processedPacketsCount;
+			double averagePacketByteSize = (processedPacketsSizeTotal * 1.0) / processedPacketsCount;
```

Comment:
Why do we multiply by 1.0 ?

Reply:
It was to have a better precision, as both `processedPacketsSizeTotal` and `processedPacketsCount` are int the result still will be the int version. I've changed it to be more exact as used double casting.